odh-manifests/devconfus2021/jupyterhub/base/jupyterhub-singleuser-profiles-sizes-configmap.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: odh-jupyterhub-sizes
+  labels:
+    jupyterhub: singleuser-profiles
+data:
+  jupyterhub-singleuser-profiles.yaml: |
+      sizes:
+      - name: Medium
+        resources:
+          requests:
+            memory: 2Gi
+            cpu: 100m
+          limits:
+            memory: 4Gi
+            cpu: 2
+      - name: Large - CPU intensive
+        resources:
+          requests:
+            memory: 4Gi
+            cpu: 1
+          limits:
+            memory: 8Gi
+            cpu: 4
+      - name: Large - Memory intensive
+        resources:
+          requests:
+            memory: 16Gi
+            cpu: 100m
+          limits:
+            memory: 32Gi
+            cpu: 4
+      - name: Extra Large - Memory intensive
+        resources:
+          requests:
+            memory: 16Gi
+            cpu: 100m
+          limits:
+            memory: 48Gi
+            cpu: 4
```

Comment:
@pacospace do you have a size preference? Let's leave just a single one in there for  workshoppers :slightly_smiling_face: